db/migrate/20170717154257_add_ticket_confirmation_to_email_settings.rb

```diff
@@ -0,0 +1,7 @@
+class AddTicketConfirmationToEmailSettings < ActiveRecord::Migration
+  def change
+    add_column :email_settings, :send_on_ticket_confirmation, :boolean, default: false
```

Comment:
You can send a different email, but the subject can always be something like _Your tickets for ConferenceName_. So I mean that you can find a default message that can be used for all conferences, so the admin doesn't need to configure this. It is also less confusing for user, who will always receive similar email.

And in this way, you don't need to store anything in the DB. :wink: